Use app.css instead of different CSS files

raghunayyar opened the pull request:
Generating a singular CSS file from grunt exactly like it works for the javascript.

@georgehrke

tcitworld commented on js/Gruntfile.js:
```diff
@@ -42,20 +42,27 @@ module.exports = function(grunt) {
 				dest: '<%= meta.productionJS %>app.js'
-			}
+			},
+            extras: {
```
Seems to be an indentation issue here.

raghunayyar replied:
weird, phpstorm needs to be configured. I will add another commit.

tcitworld replied:
I've have a lot of phpstorm indentation issues too. ;-)

Commit:
Initial Commit: Use app.css instead of different CSS files in main.php

Update main.php, corrects path.

Adds files to app folder, concats CSS to public folder.

exclude css/app from build

raghunayyar commented:
Btw, the PR is updated and is good for review.

georgehrke commented:
Can you move the existing css files to css/app/ ?

georgehrke commented:
cc @tcitworld

georgehrke commented on templates/main.php:
```diff
@@ -24,12 +24,7 @@
-	'datepicker',
-	'eventdialog',
-	'settings'
+	'../css/public/app'
```
I'm pretty sure you can just say `public/app`

raghunayyar replied:
oh lol.

georgehrke commented:
and please fix your local git config
https://github.com/owncloud/calendar/pull/660/commits
😉
